cli/src/main/java/com/devonfw/tools/ide/property/Property.java

```diff
@@ -331,13 +331,25 @@ protected V getNullValue() {
    */
   public boolean apply(CliArguments args, IdeContext context, Commandlet commandlet, CompletionCandidateCollector collector) {
 
+    return apply(this.name, args, context, commandlet, collector);
+  }
+
+  /**
+   * @param args the {@link CliArguments} already {@link CliArguments#current() pointing} the {@link CliArgument} to apply.
+   * @param context the {@link IdeContext}.
+   * @param commandlet the {@link Commandlet} owning this property.
+   * @param collector the {@link CompletionCandidateCollector}.
+   * @return {@code true} if it matches, {@code false} otherwise.
+   */
+  protected boolean apply(String normalizedName, CliArguments args, IdeContext context, Commandlet commandlet, CompletionCandidateCollector collector) {
```

Comment:
Please adjust Javadoc. normalizedName is missing.

Reply:
Done.